### ISSUE TITLE
Provide an easier way to discover and add a tool's canvas_id

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/canvas-app-explorer#177]
After #166 if a tool doesn't have a canvas_id entered it won't display on the page. This is needed to link up the add/remove functionality.

However it's difficult to figure out what these id's are.

The only way I know now is to go to either the account 
`<canvas url>/api/v1/accounts/1/tabs
`
or course
`<canvas url>/api/v1/course/123456/tabs`

And pull that id off of the external tools. Perhaps there's another API for this but I don't see them in the UI.

Minimally I think we should have a page that displays all of the ids for admins in this tool. It might be nice if it was easily accessible from the Django admin page with Canvas Id.

**Comment by jonespm** [comment]
I'm marking this low priority since this is something a developer can fill in for now, it just might be nice in the future. If we retrieve from sub-accounts we'll want to make this it's own interface probably.
